### ISSUE TITLE
Missing name column

### ISSUE THREAD

**Issue by Esfo** [stevenbenner/pacfinder#3]
Hello, I don't have a name column, it doesn't seem to be hidden in any way that I can reveal it.

Any ideas on a troubleshooting step?

![2022-03-03-125354_](https://user-images.githubusercontent.com/53882979/156624248-aa098305-86dd-41c4-b7d8-a3c46fc174ec.png)

**Comment by stevenbenner** [comment]
Thanks for the bug report. Seeing the screenshot is helpful, though the mystery persists.

May I ask you to check the following two items:

1. **Check what column widths PacFinder believes it has set**

    Run this command, and let me know the output:
  `gsettings get 'com.stevenbenner.pacfinder' 'package-list-column-widths'`

2. **Check for error/warning messages**

    Run PacFinder from the command line (`pacfinder`), and see if it prints out any messages.

**Comment by Esfo** [comment]
apologies, I thought I ran this from the command line before and saw nothing but it turns out that's not true

```
me@hi ~ % gsettings get 'com.stevenbenner.pacfinder' 'package-list-column-widths'
(0, 788, 168, 2744)
me@hi ~ % pacfinder

(pacfinder:258498): Gtk-WARNING **: 15:12:07.867: Negative content width -12 (allocation 1, extents 6x7) while allocating gadget (node button, owner GtkButton)

(pacfinder:258498): Gtk-WARNING **: 15:12:07.891: Negative content width -12 (allocation 1, extents 6x7) while allocating gadget (node button, owner GtkButton)

(pacfinder:258498): Gtk-WARNING **: 15:12:07.923: Negative content width -12 (allocation 1, extents 6x7) while allocating gadget (node button, owner GtkButton)

(pacfinder:258498): Gtk-WARNING **: 15:12:07.944: Negative content width -12 (allocation 1, extents 6x7) while allocating gadget (node button, owner GtkButton)

(pacfinder:258498): Gtk-WARNING **: 15:12:07.965: Negative content width -12 (allocation 1, extents 6x7) while allocating gadget (node button, owner GtkButton)

(pacfinder:258498): Gtk-WARNING **: 15:12:07.986: Negative content width -12 (allocation 1, extents 6x7) while allocating gadget (node button, owner GtkButton)
```

**Comment by stevenbenner** [comment]
Ok, good info. That first zero is definitely wrong, so let's set that to something more reasonable.

Make sure application is closed then run the following command:

`gsettings set 'com.stevenbenner.pacfinder' 'package-list-column-widths' '(400, 788, 168, 2344)'`

Open PacFinder again and see if the column is now visible.

Those warnings may or may not be related to the column width being set to zero. It would be interesting to see if they go away after setting the width.

**Comment by Esfo** [comment]
They did go away, and that fixed it.

**Comment by stevenbenner** [comment]
Glad to hear that resolved the problem.

I'm not sure how it got set to zero width, but that is a situation which the software should prevent from happening because it's basically impossible to recover from without knowing the magic gsettings command to fix it.

I'll add a check to the code to enforce a minimum value on the saved column widths. That should prevent this from happening in the future.

**Comment by stevenbenner** [comment]
This problem should be fixed (or at least mitigated) with commit 7df7148e88b56b29c53f93fc928fde69d6f72974.

I have added a check that will enforce a minimum value of 50 pixels on saved package list column widths.

This fix will be included in the next PacFinder release.
